apps/docs/app/cla/page.tsx

```diff
@@ -8,7 +8,7 @@ export default async function ClaPage() {
 
 	return (
 		<>
-			<Header activeId={null} />
```

Comment:
What was activeId doing before? I can't figure it out.

Reply:
nothing! :P

Reply:
it _was_ being used in another page (maybe ArticleDocsPage?) but it was copy-pasted over i think needlessly